frontend/internal-packages/agent/src/chat/workflow/nodes/analyzeRequirementsNode.ts

```diff
@@ -1,7 +1,19 @@
+import * as v from 'valibot'
+import { PMAgent } from '../../../langchain/agents'
+import type { BasePromptVariables } from '../../../langchain/utils/types'
+import { convertSchemaToText } from '../../../utils/convertSchemaToText'
 import type { WorkflowState } from '../types'
 
 const NODE_NAME = 'analyzeRequirementsNode'
 
+const requirementsAnalysisSchema = v.object({
+  brd: v.string(),
```

Comment:
It does not seem necessary to make this the only abbreviation. I thought `businessRequirement` would be fine, what do you think?